test/sol5/kyberStorage.js

```diff
@@ -298,26 +347,63 @@ contract('KyberStorage', function(accounts) {
+            // set up 1 mock reserve and 1 fpr reserve, 1 with fee and 1 not
+            await kyberStorage.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: admin});
+            let result = await nwHelper.setupReserves({address: network}, [token], 1,1,0,0, accounts, admin, operator);
+            let reserveInstances = result.reserveInstances;
+            let reserveAddresses= [];
+            let reserveIds = [];
+            let reserveFeeData = [];
+            // add all reserve to network
+            for (const value of Object.values(reserveInstances)) {
+                let reserve = value;
+                if (reserve.type = "TYPE_MOCK") {
+                    reserve.onChainType = ReserveType.UTILITY;
+                }
+                await kyberStorage.addReserve(reserve.address, reserve.reserveId, reserve.onChainType, {from: network});
+                reserveAddresses.push(reserve.address);
+                reserveIds.push(reserve.reserveId);
+                reserveFeeData.push(reserve.type != "TYPE_MOCK");
+
+                assert(await kyberStorage.convertReserveAddresstoId(reserve.address) == reserve.reserveId, "unexpected reserveId");
+                assert(await kyberStorage.convertReserveIdToAddress(reserve.reserveId) == reserve.address, "unexpected reserveId");
+                let reserveData = await kyberStorage.getReserveDetailsById(reserve.reserveId);
+                assert(reserveData.reserveAddress == reserve.address, "unexpected reserve address");
+                assert(reserveData.resType == reserve.onChainType, "unexpected reserve on chain type");
+                assert(reserveData.isFeeAccountedFlags == (reserve.type != "TYPE_MOCK"), "unexpected fee accounted flag");
+            }
+            Helper.assertEqualArray(await kyberStorage.getReserves(), reserveAddresses, "unexpected reserve addresses");
+            Helper.assertEqualArray(await kyberStorage.convertReserveAddressestoIds(reserveAddresses), reserveIds);
+            Helper.assertEqualArray(await kyberStorage.convertReserveIdsToAddresses(reserveIds), reserveAddresses);
+            Helper.assertEqualArray(await kyberStorage.getFeeAccountedData(reserveIds), reserveFeeData);
+        });
     });
 
     describe("test listing token pair and removing reserve", async() => {
         before("deploy and setup kyberStorage instance & add 1 mock reserve, & 1 mock token", async() => {
             kyberStorage = await KyberStorage.new(admin);
             await kyberStorage.setNetworkContract(network, {from: admin});
             await kyberStorage.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: admin});
-            //init 1 mock reserve
-            let result = await nwHelper.setupReserves(network, [], 1,0,0,0, accounts, admin, operator);
+            //init 2 mock reserve
+            let result = await nwHelper.setupReserves(network, [], 2,0,0,0, accounts, admin, operator);
```

Comment:
This is for coverage at line https://github.com/KyberNetwork/smart-contracts/blob/bdcf45724c32115ce2d7ce66bb0461be1fd82ab6/contractsSol5/KyberStorage.sol#L363